.node-version

```diff
@@ -1 +1 @@
-14.5.0
+14.17.3
```

Comment:
This is the version currently used in Along, and one day we'll use it in Learning Platform.